HunabKu_kamunu/hunabku_kamunu/endpoints/kamunu_plugin.py

```diff
@@ -53,13 +53,13 @@ def search_organizations(self):
 
         @apiExample {curl} Example usage:
             # Get dehydrated document of the organization.
-            curl -i http://apis.colav.co/organizations?apikey=colavudea&query=Universidad%20de%20antioquia&return=Dehydrated_document
+            curl -i http://apis.colav.co/organizations?apikey=XXXX&query=Universidad%20de%20antioquia&return=Dehydrated_document
```

Comment:
we need to do something with this, because the test apikey is public now

Reply:
Sure, that was fixed in the last confirmation.